examples/index.js

```diff
@@ -34,13 +41,15 @@ let turnCount = 0;
 let winState = WIN_STATES.NO_WINNER;
 
 // This is the official starting state of the game. Everything is blank.
-let gameState = [
+const STARTING_STATE = [
     // Column 0          Column 1            Column 2
     [squareStates.BLANK, squareStates.BLANK, squareStates.BLANK], // Horizontal row 0
     [squareStates.BLANK, squareStates.BLANK, squareStates.BLANK], // Horizontal row 1
     [squareStates.BLANK, squareStates.BLANK, squareStates.BLANK], // Horizontal row 2
 ];
 
+let gameState = STARTING_STATE;
+
```

Comment:
We should make the game state a deep copy of the starting state when initializing the game state
https://www.freecodecamp.org/news/how-to-clone-an-array-in-javascript-1d3183468f6a/#:~:text=8.%20JSON.parse%20and%20JSON.stringify%20(Deep%20copy)